src/lib/prisma.ts

```diff
@@ -23,9 +23,18 @@ const tCO2e = 'tCO2e'
 
 export async function upsertScope1(
   emissions: Emissions,
-  scope1: OptionalNullable<Omit<Scope1, 'id' | 'metadataId' | 'unit'>>,
+  scope1: OptionalNullable<Omit<Scope1, 'id' | 'metadataId' | 'unit'>> | null,
   metadata: Metadata
 ) {
+  if (scope1 === null) {
+    if (emissions.scope1Id) {
+      await prisma.scope1.delete({
+        where: { id: emissions.scope1Id },
+      })
+    }
+    return null
+  }
+
```

Comment:
I've thought about this. Perhaps we should use explicit `DELETE` requests instead? That way, we remove the possibility of deleting data by accident.

It's not that big of a difference between `null` and `undefined`

Reply:
We discussed this and we will come back to this later.